helm/values.yaml

```diff
@@ -47,72 +55,59 @@ validators:
   #   password: notarypass
   #   # or
   #   secret_name: notarysecret  # define a k8s secret with `username` and `password` fields
-- name: docker_essentials  # public notary form dockerhub
+# pre-configured nv1 validator for public notary from Docker Hub
+- name: dockerhub_essentials
   type: notaryv1
   host: notary.docker.io
-  pub_keys:
-  # this is the public key for all official dockerhub images (e.g. redis)
+  trust_roots:
+    # public key for official docker images (https://hub.docker.com/search?q=&type=image&image_filter=official)
   - name: docker_official
     key: |
-      -----BEGIN PUBLIC KEY-----
+      -----BEGIN PUBLIC key-----
       MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEOXYta5TgdCwXTCnLU09W5T4M4r9f
       QQrqJuADP6U7g5r9ICgPSmZuRHP/1AYUfOQW3baveKsT969EfELKj1lfCA==
-      -----END PUBLIC KEY-----
-  # this is the public key to the `securesystemsengineering` repo on dockerhub
+      -----END PUBLIC key-----
+  # public key for Connaisseur images
```

Comment:
That is not true is it? If we deploy other images in the `securesystemsengineering` repo they'd be signed by the same key right? (Which is ok as we discussed but I'd change the comment back to the repo wording)